tools/packaging/build_wheel.py

```diff
@@ -16,11 +17,12 @@
     level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
 )
 logger = logging.getLogger(__name__)
-logger.setLevel(logging.DEBUG)
+logger.setLevel(logging.INFO)
```

Comment:
seems unrelated

Reply:
It's unrelated but also benign since this file isn't even used yet.